sdks/python/apache_beam/ml/inference/api.py

```diff
@@ -17,16 +17,17 @@
 # mypy: ignore-errors
 
 from dataclasses import dataclass
-from typing import Tuple
-from typing import TypeVar
-from typing import Union
+from typing import Callable
 
 import apache_beam as beam
 from apache_beam.ml.inference import base
+from apache_beam.typehints import Tuple
```

Comment:
I asked @robertwb about this in https://github.com/apache/beam/pull/17762.  He thought that we should prefer native types.  Does that sound right?

Reply:
Removed.